src/Microsoft.AspNetCore.ResponseCompression/ResponseCompressionProvider.cs

```diff
@@ -38,7 +38,13 @@ public ResponseCompressionProvider(IServiceProvider services, IOptions<ResponseC
             if (_providers.Length == 0)
             {
                 // Use the factory so it can resolve IOptions<GzipCompressionProviderOptions> from DI.
-                _providers = new ICompressionProvider[] { new CompressionProviderFactory(typeof(GzipCompressionProvider)) };
+                _providers = new ICompressionProvider[]
+                {
+                    new CompressionProviderFactory(typeof(GzipCompressionProvider)),
+#if NETCOREAPP2_1
+                    new CompressionProviderFactory(typeof(BrotliCompressionProvider))
+#endif
```

Comment:
List all TFMs and include the else and error cases so we know if these need updating.

Reply:
Agreed on the error case, but what should the other TFMs do?

Reply:
The other TFMs can be empty.